### DISCUSSION TITLE
Parameter data_divide_ratio in input.yaml

### DISCUSSION THREAD

**Discussion by AugustinLu** [MDIL-SNU/SevenNet#130] [General]
Hello,

When I was using SevenNet v0.9.3, I used to set parameter `data_divide_ratio` in the yaml file to split the original dataset into training and validation sets before training a model.
Example: `data_divide_ratio: 0.2`

However, after updating to v0.10.2 recently, it seems that this variable seems to no longer have any effect, and a message reporting the absence of validset appears: 
`Failed to get error recorder key: TotalLoss or validset is missing. There will be no best checkpoint.`

Is the parameter `data_divide_ratio`  still in use? 

Thank you very much.

**Comment by YutackPark** [comment]
Sorry for late answer. It is temporarily not working in v0.10.x. I'll make it back as soon as possible. If you want previous training mode with that working that key, you may able to use `sevenn -m train_v1` for it. As I didn't tested routine for this legacy mode, use with care.

**Comment by AugustinLu** [comment]
Thank you for your kind reply.

I understand the situation. For now, we will try either with ` train_v1`, or prepare the training, validation and test sets beforehand.

**Reply by YutackPark** [reply]
The feature is included in official v0.10.2. Check when you have a time!

**Reply by AugustinLu** [reply]
I have confirmed that it is working with the latest revision.

Thank you for your swift actions!
